packages/react-scripts/package.json

```diff
@@ -61,8 +60,8 @@
     "strip-ansi": "3.0.1",
     "style-loader": "0.13.1",
     "url-loader": "0.5.7",
-    "webpack": "1.14.0",
-    "webpack-dev-server": "1.16.2",
+    "webpack": "2.2.1",
+    "webpack-dev-server": "2.3.0",
     "webpack-manifest-plugin": "1.1.0",
     "whatwg-fetch": "2.0.2"
   },
```

Comment:
Should we include the exclusion of "/scripts" folder in jest ignore patterns here since it tries to run `test.js`?

```"testPathIgnorePatterns": [
      "<rootDir>[/\\\\](build|docs|node_modules|scripts)[/\\\\]"
],```